### ISSUE TITLE
Symfony Framework Kernel should use $this->getProjectDir() instead of ../ for config/

### ISSUE THREAD

**Issue by 7ochem** [symfony/recipes#784]
The Symfony Framework Kernel should use `$this->getProjectDir()` to locate the project root and access config files from there instead of `../config/` which is being used now since Symfony 5.1.

In 4.4 (4.2) Kernel:
https://github.com/symfony/recipes/blob/81543c64382d0a8898479182e18b460ef196ab15/symfony/framework-bundle/4.2/src/Kernel.php#L38-L43

In 5.1 Kernel:
https://github.com/symfony/recipes/blob/81543c64382d0a8898479182e18b460ef196ab15/symfony/framework-bundle/5.1/src/Kernel.php#L16-L19

In all other places (`MicroKernelTrait` and `\Symfony\Component\HttpKernel\Kernel`) `$this->getProjectDir()` is being used consistently.

**Comment by nicolas-grekas** [comment]
There is no need to use getProjectDir, all is fine with `..` now, the path is resolved relative to the Kernel's file.

**Comment by 7ochem** [comment]
I have one project where we have placed the Kernel.php file into `src/Framework/`. Before 5.1 this worked just fine. Now I have updated to 5.1, and ran `symfony:recipes:install`, moved `src/Kernel.php` to `src/Framework/Kernel.php` and ran into this issue.

Like described in the text above, before 5.1 it used `getProjectDir()` and in other places, this is also used. Why was this changed in the `Kernel.php` file overnight?
